samples/subsys/display/cfb_custom_font/src/main.c

```diff
@@ -11,23 +11,20 @@
 
 #include "cfb_font_dice.h"
 
-#if defined(CONFIG_SSD1306)
-#define DISPLAY_NODE DT_INST(0, solomon_ssd1306fb)
-#elif defined(CONFIG_SSD16XX)
-#define DISPLAY_NODE DT_INST(0, solomon_ssd16xxfb)
-#else
-#error Unsupported board
-#endif
-
 void main(void)
 {
-	const struct device *display = DEVICE_DT_GET(DISPLAY_NODE);
+	const struct device *display = DEVICE_DT_GET(DT_CHOSEN(zephyr_display));
 	int err;
 
 	if (!device_is_ready(display)) {
 		printk("Display device not ready\n");
 	}
 
+	if (display_set_pixel_format(display, PIXEL_FORMAT_MONO10) != 0) {
```

Comment:
This appears to be an unrelated change for this commit; what's going on here?

Reply:
sloppiness, will update after https://github.com/zephyrproject-rtos/zephyr/pull/40876 is merged.

Reply:
#40876 doesn't seem to have anything to do with this; is that the right link?

Reply:
yes, if I push now, CI will fail again because of that.